products/icinga2.md

```diff
@@ -0,0 +1,138 @@
+
+# eol(x) = releaseDate(x+2)
+# support(x) = releaseDate(x+1)
+releases:
+-   releaseCycle: "2.14"
+    releaseDate: 2023-07-12
+    support: true
+    eol: false
+    latest: "2.14.1"
+    latestReleaseDate: 2023-12-20
+
+-   releaseCycle: "2.13"
+    releaseDate: 2021-08-02
+    support: 2023-07-12
+    eol: false
+    latest: "2.13.9"
+    latestReleaseDate: 2023-12-20
+
+-   releaseCycle: "2.12"
+    releaseDate: 2020-08-03
+    support: 2021-08-02
+    eol: 2023-07-12
+    latest: "2.12.10"
+    latestReleaseDate: 2023-02-16
+
+-   releaseCycle: "2.11"
+    releaseDate: 2019-09-19
+    support: 2020-08-03
+    eol: 2021-08-02
+    latest: "2.11.11"
+    latestReleaseDate: 2021-08-18
+
+-   releaseCycle: "2.10"
+    releaseDate: 2018-10-11
+    support: 2019-09-19
+    eol: 2020-08-03
+    latest: "2.12.10"
+    latestReleaseDate: 2023-02-16
+
+-   releaseCycle: "2.9"
+    releaseDate: 2018-07-17
+    support: 2018-10-11
+    eol: true
+    latest: "2.9.3"
+    latestReleaseDate: 2019-07-30
+
+-   releaseCycle: "2.8"
+    releaseDate: 2017-11-16
+    support: 2018-07-17
+    eol: true
+    latest: "2.8.4"
+    latestReleaseDate: 2018-04-25
+
+-   releaseCycle: "2.7"
+    releaseDate: 2017-08-02
+    support: 2017-11-16
+    eol: true
+    latest: "2.7.2"
+    latestReleaseDate: 2017-11-09
+
+-   releaseCycle: "2.6"
+    releaseDate: 2016-12-13
+    support: 2017-08-02
+    eol: true
+    latest: "2.6.3"
+    latestReleaseDate: 2017-03-29
+
+-   releaseCycle: "2.5"
+    releaseDate: 2016-08-22
+    support: 2016-12-13
+    eol: true
+    latest: "2.5.4"
+    latestReleaseDate: 2016-08-30
+
+-   releaseCycle: "2.4"
+    releaseDate: 2015-11-16
+    support: 2016-08-22
+    eol: true
+    latest: "2.4.10"
+    latestReleaseDate: 2016-05-19
+
+-   releaseCycle: "2.3"
+    releaseDate: 2015-03-09
+    support: 2015-11-16
+    eol: true
+    latest: "2.3.11"
+    latestReleaseDate: 2015-10-19
+
+-   releaseCycle: "2.2"
+    releaseDate: 2014-11-17
+    support: 2015-03-09
+    eol: true
+    latest: "2.2.4"
+    latestReleaseDate: 2015-02-05
+
+-   releaseCycle: "2.1"
+    releaseDate: 2014-08-29
+    support: 2014-11-17
+    eol: true
+    latest: "2.1.1"
+    latestReleaseDate: 2014-09-16
+
+-   releaseCycle: "2.0"
+    releaseDate: 2014-06-16
+    support: 2014-08-29
+    eol: true
+    latest: "2.0.2"
+    latestReleaseDate: 2014-08-07
+
+-   releaseCycle: "0.0"
+    releaseDate: 2012-10-19
+    support: 2014-06-16
+    eol: true
+    latest: "0.0.11"
+    latestReleaseDate: 2014-05-16
+
+---
+
+> [Icinga2](https://icinga.com/docs/icinga-2/latest/doc/01-about/) is an open-source monitoring server, is designed to monitor various aspects  
+> of IT infrastructure, including networks, servers, virtual machines, and cloud services.
+
+Since 2.10, Icinga2 follows an N-1 support strategy  
+(meanings that the 2 most recent major versions receive security and bug fixes)  
```

Comment:
> Since 2.10, Icinga2 follows an N-1 support strategy  

before 2.10, release are in EOL = true

I base this information on the release date of major versions and their latest available versions.
From v2.10, we see that version N-1 is still supported even after the release of version N

Reply:
The "Software Versions" paragraph on https://icinga.com/subscriptions/support-matrix/ seems to imply that latest version is supported with "Full Updates" and the N-1 version with "Security Updates", which is basically what you documented if I am not mistaken as the after 2.10 rule.

And what you mean by this comment is that before 2.10 only the latest release was supported ?

Anyway if that's not documented somewhere on https://icinga.com/ it should be clearly stated that this is a guess in the product description.

Reply:
This is clearly documented in the _Software Versions_ paragraph of https://icinga.com/subscriptions/support-matrix/.
For both Icinga and Icinga Web the latest version receives "Full Updates", while the previous one receives "Security Updates".